README.md

```diff
@@ -162,6 +162,16 @@ fmt.Println(jsonFeed.HomePageURL)
 
 ## Advanced Usage
 
+#### Instanciate Parser with BasicAuthentication:
```

Comment:
Spelling: Instantiate

Reply:
Oh, sorry i fix it :)

Reply:
Let me know if you are able to take a look at this.

Reply:
Hello, of course I correct this tonight